### ISSUE TITLE
Documentation basic usage missing a get_env example

### ISSUE THREAD

**Issue by cglacet** [aio-libs/aiohttp-jinja2#269]
I've spent the last 10 minutes looking for how to add a global to jinja2 environment ([get_env](https://aiohttp-jinja2.readthedocs.io/en/stable/#aiohttp_jinja2.get_env), [code](https://github.com/aio-libs/aiohttp-jinja2/blob/master/aiohttp_jinja2/__init__.py#L39)).

For example, adding the `zip` function:

```python
env = aiohttp_jinja2.get_env(app)
env.globals.update(zip=zip)
```

It appears to me as basic and that would be nice to have it referenced, but maybe I'm wrong ^^

**Comment by asvetlov** [comment]
Feel free to make a pull request

**Comment by asvetlov** [comment]
The change is published in 1.1.2 release.
